cli_scripts/voice_coding_cli.py

```diff
@@ -0,0 +1,26 @@
+import subprocess,sys,click
+
+@click.command()
+@click.option('--voice/--no-voice', default = False, help = "Use --voice if you want to code using your voice.")
+@click.option('--speech', prompt = "What is your default Speech Recognition engine ? ", help = "Choose your default Speech Recognition engine")
+def enable_voice_coding(voice,speech):
+    """Welcome to LarynxCode. This is a simple CLI that enables you to allow voice programming on your editor."""
+    if voice:
+        if speech == 'wsr':
+            cmds = [b'cd ..\..\..',b'c:',b'cd Caster',b'python -m dragonfly load --engine sapi5inproc _*.py --no-recobs-messages']
+            encoding = 'latin1'
+            p = subprocess.Popen(
+                'cmd.exe',
+                stdin = subprocess.PIPE,
+                stdout = subprocess.PIPE,
+                stderr = subprocess.PIPE
+            )
+            for cmd in cmds:
+                p.stdin.write(cmd + b"\n")
+            p.stdin.close()
+            click.echo(p.stdout.read())
+        else:
```

Comment:
Don't give users the (fake) option if you don't support other options. Instead, just say somewhere in the docs that you only support wsr.

Reply:
Noted. Will remove the other --speech flag then.
Actually, kept it coz I wanted to turn this project into a full-fledged CLI application that supports DNS. But that can be added anytime.